packages/admin/dashboard/src/routes/price-lists/common/components/price-list-customer-group-rule-form/price-list-customer-group-rule-form.tsx

```diff
@@ -59,9 +59,12 @@ export const PriceListCustomerGroupRuleForm = ({
     prefix: PREFIX,
   })
   const { customer_groups, count, isLoading, isError, error } =
-    useCustomerGroups(searchParams, {
-      placeholderData: keepPreviousData,
-    })
+    useCustomerGroups(
+      { ...searchParams, fields: "*customers.id" },
```

Comment:
question: Do we need the `*` here? Would have expected this to just be `customers.id`

Reply:
yes, fixed!